internal/util/mock_mounter_test.go

```diff
@@ -0,0 +1,98 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Did you use the mock to have it tested without the OS? Maybe it is more natural to write a real Linux-only test.

Reply:
Tests that modify the underlying system are extremely obnoxious and tend to be flaky. I'd prefer to just keep the mock test for now. This still gets tested by the integration tests.

Reply:
> This still gets tested by the integration tests

Ok, that answers my concerns 👍